src/ir/proto/sql/decode_test.cc

```diff
@@ -58,4 +58,47 @@ INSTANTIATE_TEST_SUITE_P(
     DecodeSourceTableColumnTest, DecodeSourceTableColumnTest,
     testing::ValuesIn(kSourceColumnPaths));
 
+class DecodeLiteralTest :
+    public testing::TestWithParam<absl::string_view> {};
+
+TEST_P(DecodeLiteralTest, DecodeLiteralTest) {
+  absl::string_view literal_str = GetParam();
+  std::string textproto = absl::StrFormat(R"(literal_str: "%s")", literal_str);
+  Literal literal_proto;
+
+  ASSERT_TRUE(google::protobuf::TextFormat::ParseFromString(
+      textproto, &literal_proto))
+      << "Could not decode Literal";
+
+  IRContext ir_context;
+  DecoderContext decoder_context(ir_context);
+  ir::Value result = DecodeLiteral(literal_proto, decoder_context);
+  const value::OperationResult *op_result = result.If<value::OperationResult>();
+  EXPECT_THAT(op_result, testing::NotNull());
+  EXPECT_EQ(op_result->name(), "out");
+  const Operation &operation = op_result->operation();
+  EXPECT_THAT(operation.parent(), testing::IsNull());
```

Comment:
Adding using declarations like `using testing::isNull;` in this file could help readability. e.g., 
```C++
EXPECT_THAT(operation.parent(), IsNull());
```
instead of
```C++ 
EXPECT_THAT(operation.parent(), testing::IsNull());
```

Reply:
Good point, will do.

Reply:
Done.